tests/integration/gen_regression_output.py

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright 2019 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-module-docstring
+# pylint: disable=missing-function-docstring
+
+from tests.integration.infra import generate_known_regression_output
```

Comment:
That's a fair point. My reasoning as you guessed is that this is only used for testing now. Not only that, but it is vital for testing, because really without the output created from this script and the infra logic, the tests simply cannot run.

Would you be ok with me leaving it as is for now? I expect at least some refactoring of this suite in the near future where we'll have written more commands that probably need a better crash-dump and we'll need to change the infrastructure so it won't use hardcoded values.

Reply:
Yea, I'm good as is.